Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -1684,27 +1685,18 @@
   </Target>
   <PropertyGroup>
     <PostBuildEvent>
-      copy /y $(OutDir)..\WebRole\Microsoft.NodejsTools.WebRole.dll $(OutDir)Debugger\RemoteDebug\
+      copy /y $(OutDir)..\WebRole\Microsoft.NodejsTools.WebRole.dll $(OutDir)RemoteDebug\
```

Comment:
This won't copy the assembly to the VSIX (necessary for git deploy scenarios, where the file must be included in the project manually)